components/org.wso2.carbon.analytics.msf4j.interceptor.common/pom.xml

```diff
@@ -69,6 +73,7 @@
         <import.package>
             org.wso2.carbon.config.*;version="${carbon.config.version.range}",
             org.wso2.carbon.analytics.idp.client.*;version="${carbon.analytics-common.version.range}",
+            org.wso2.carbon.stream.processor.common.*;version="0.0.0",
```

Comment:
Please, please, please Don't put OSGi import version as `0.0.0`.
This might lead to some obscure OSGi class loading issue, and trust me they are very hard to solve  :worried: 

@Niveathika92 Shall we add a Maven property `carbon.stream.processor.common.version.range` property with the proper versions range and use it here.